src/SeoToolkit.Umbraco.MetaFields.Core/Notifications/AfterMetaTagsNotification.cs

```diff
@@ -7,11 +9,20 @@ public class AfterMetaTagsNotification : INotification
     {
         public string ContentTypeAlias { get; }
         public MetaTagsModel MetaTags { get; }
+        public IPublishedContent Content { get; set; }
```

Comment:
I just saw that I allowed `set` on the automatic property.  This should probably be updated to `get` only, like the other properties.  Just habit (and auto-complete).

Reply:
PR #199 to fix

Reply:
(I don't use GitHub PRs very often.  I couldn't see a way to amend or reopen this one once it was merged.  Let me know if there was an easier way to do this.  A second PR feels like overkill)